### PR TITLE
Multiple inhand sprites fix

### LINKED ISSUE #28043: dragnet has no inhand

### LINKED ISSUE #29096: one-point bluespace-gravitational manipulator has no inhand
what it says on the tin

### LINKED ISSUE #29254: chem sprayer has no inhand

### PR DESCRIPTION
**Pull request by kingofkosmos**
Fixes One-point bluespace-gravitational manipulator's inhand.
Adds inhand sprites for the Chem Sprayer & DRAGnet. (Thanks to @FuryMcFlurry for the sprites!)

Fixes #29096 
Fixes #29254 
Fixes #28043 

:cl:
imageadd: Inhands for One-point bluespace-gravitational manipulator, DRAGnet and Chem Sprayer.
/:cl:

~~Also adds placeholder-inhands for all the severed limbs - a gray block, same as a pipe, but it's better than nothing. If someone wants to make better inhands for the limbs, I'll add them here.~~
~~Still waiting for the inhand sprites for:~~
* ~~Chem Sprayer~~
* ~~DRAGnet~~
~~Will add once someone makes them (already requested on the forums).~~

### CODE REVIEW

File: code/modules/surgery/bodyparts/bodyparts.dm [hunk 1]
```diff
@@ -2,6 +2,7 @@
 /obj/item/bodypart
 	name = "limb"
 	desc = "why is it detached..."
+	item_state = "buildpipe"
```
Comment (RemieRichards) [inline code/modules/surgery/bodyparts/bodyparts.dm]: No.
Flesh limbs should not show up as pipes, that's dumb.

**Comment by DoctoreMad** [comment]
>Also adds placeholder-inhands for all the severed limbs - a gray block, same as a pipe

Well that's not really necessary. If we get proper limb sprites we can use them, if we don't have them, its not like its some sort of lethal item or anything so why do you need to change it?

**Comment by kingofkosmos** [comment]
>Well that's not really necessary. If we get proper limb sprites we can use them, if we don't have them, its not like its some sort of lethal item or anything so why do you need to change it?

I thought that anything in hand would add to the "feel" of the game. If people disagree I can remove it from the PR and wait for proper inhands if there will be any.

**Comment by optimumtact** [comment]
it's really quite remarkable that dismembered limbs got in without an inhand

**Comment by RemieRichards** [comment]
Not really, at the time we had poor support for colour inhands (read: none), no support for overlays on inhands, etc.

Not to mention they don't DO anything, they're not weapons and don't specifically -need- an inhand, certainly not enough to justify a grey pipe.

**Comment by AnturK** [comment]
Yeah, just skip the inhands for now , we'll just wait bit longer.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Do the thing remie wants

**Comment by FuryMcFlurry** [comment]
I've contributed dragnet and chemsprayer inhands. Awaiting change.

**Comment by Tacolizard** [comment]
pics or GTFO

**Comment by Cyberboss** [comment]
@kingofkosmos

**Comment by FuryMcFlurry** [comment]
@Tacolizard 
image

guess which one is which

**Comment by kingofkosmos** [comment]
Original post edited. Added Fury's DRAGnet and Chem Sprayer inhands.

Removed severed limbs stuff. Unless someone wants to make them, I'll add them to this PR. Btw should there be an inhand for every limb? Or just for heads and a some sort of blob-lookalike for all the other limbs + chest?

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**